config/services.yaml

```
@@ -16,4 +16,7 @@ services:
     default.converter.with.cache:
         abstract: true
         class: Neusta\ConverterBundle\Converter\DefaultCachedConverter
+        decorates: default.converter
+        arguments:
+            $inner: '@default.converter'
```

```suggestion
            $inner: '@.inner'
```

See: https://symfony.com/doc/5.4/service_container/service_decoration.html